.kitchen.yml

```diff
@@ -54,3 +54,70 @@ suites:
   - name: ftp
     run_list:
       - recipe[osl-jenkins::ftp]
+
+  # In order to properly test GitHub integration, you must export the
+  # GITHUB_USER and GITHUB_TOKEN env vars in your shell before you converge
+  # this suite.  You'll need to use an account/token that has permission to
+  # pull/push private repos and create webhooks.  You also need to choose a
+  # random TRIGGER_TOKEN string.  This is what GitHub will use to trigger
+  # Jenkins jobs remotely.
+  #
+  # For example:
+  #   $ export GITHUB_USER=examplegithubuser
+  #   $ export GITHUB_TOKEN=examplegithubtoken
+  #   $ export TRIGGER_TOKEN=exampletriggertoken
+  #   $ kitchen converge cookbook-uploader-centos-72
+  #
+  # It's worth noting that security (i.e. logging in to Jenkins) is not enabled
+  # by default, and so remotely triggered build configuration will not appear
+  # when configuring jobs.  It will, however, still be set.  If you enable
+  # user/pass security, then the Jenkins cookbook will not be able to update
+  # jobs, since it then has to log in.  To fix this, you must create a user
+  # account for the cookbook and export the JENKINS_USER and JENKINS_PASS env
+  # vars in addition to the others.
+  - name: cookbook-uploader
+    encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
+    run_list:
+      - recipe[osl-jenkins::master]
+      - recipe[jenkins::java]
+      - recipe[jenkins::master]
+      - recipe[certificate::manage_by_attributes]
+      - recipe[osl-jenkins::haproxy]
+      - recipe[osl-haproxy::default]
+      - recipe[osl-jenkins::cookbook_uploader]
```

Comment:
They are actually included in the recipe. I think this was added before I moved the recipes from the role to the `master` recipe.